### PR TITLE
Chapter 02 -2 02_2_Configurando_Bitcoin_Core_Otros.md

### PR DESCRIPTION
**Pull request by javiervargas**

### CODE REVIEW

File: es/02_2_Configurando_Bitcoin_Core_Otros.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* [Usando GordianNode-macOS](https://github.com/BlockchainCommons/GordianNode-macOS). * Si tienes una Mac moderna, puedes usar la aplicación *GordianNode* de 
+Blockchain Commons, con tecnología *BitcoinStandup*, para instalar un nodo completo en su Mac.
+* [Uso de otros scripts de Bitcoin Standup](https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts). * Blockchain Commons también ofrece una versión del 
+script de Linode que se uso anteriormente y que se puede ejecutar desde la línea de comandos en cualquier máquina Debian o Ubuntu. Este tiende a ser un script 
+de vanguardia, lo que significa que es más probable que presente nuevas funciones, como la instalación de Lightning.
+* [Configuración de un nodo Bitcoin en AWS](https://wolfmcnally.com/115/developer-notes-setting-up-a-bitcoin-node-on-aws/). * @Wolfmcnally ha escrito un tutorial 
+paso a paso para configurar Bitcoin-Core con Amazon Web Services (AWS).
+* [Configuración de un nodo Bitcoin en una Raspberry Pi 3](https://medium.com/@meeDamian/bitcoin-full-node-on-rbp3-revised-88bb7c8ef1d1). * Damian Mee explica
```
Comment (csralvall) [inline es/02_2_Configurando_Bitcoin_Core_Otros.md:17]: Add `* *` to beginning of link and remove space  after it to display italics.

File: es/02_2_Configurando_Bitcoin_Core_Otros.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* [Configuración de un nodo Bitcoin en AWS](https://wolfmcnally.com/115/developer-notes-setting-up-a-bitcoin-node-on-aws/). * @Wolfmcnally ha escrito un tutorial 
+paso a paso para configurar Bitcoin-Core con Amazon Web Services (AWS).
+* [Configuración de un nodo Bitcoin en una Raspberry Pi 3](https://medium.com/@meeDamian/bitcoin-full-node-on-rbp3-revised-88bb7c8ef1d1). * Damian Mee explica 
+cómo configurar una Nodo completo de forma sencilla en una Raspberry Pi 3.
+
+## ¿Que sigue?
+
+A menos que desee volver a una de las otras metodologías para crear un nodo Bitcoin-Core, debe:
+
+* Continúe con "bitcoin-cli" con el [Capítulo tres: Entendiendo la configuración de Bitcoin](03_0_Entendiendo_su_configuracion_Bitcoin.md).
```
Comment (csralvall) [inline es/02_2_Configurando_Bitcoin_Core_Otros.md:24]: From list of chapters in LEEME.md: `[3.0: Entendiendo Su Configuración Bitcoin](03_0_Entendiendo_Su_Configuracion_Bitcoin.md)`

**Review by csralvall — CHANGES_REQUESTED** [review]

File: es/02_2_Configurando_Bitcoin_Core_Otros.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# 2.2: Configuración de una máquina Bitcoin-Core a través de otros medios
```
[the remark this answers is not in the document]
Comment (javiervargas) [reply es/02_2_Configurando_Bitcoin_Core_Otros.md:1]: Las mayúsculas no van.

File: es/02_2_Configurando_Bitcoin_Core_Otros.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+La sección anterior, [§2.1: Configuración de un VPS Bitcoin-Core con Bitcoin Standup](02_1_Configurando_un_VPS_Bitcoin-Core_StackScript.md), presumía que 
+estaría creando un nodo completo en un VPS usando un Linode Stackscript. Sin embargo, puede crear una instancia de Bitcoin-Core a través de cualquier metodología de
+su elección y seguir los pasos posteriores de este tutorial.
+
+A continuación, se muestran otras metodologías de configuración que conocemos:
+
+* [Compilación desde la fuente] (A2_0_Compiling_Bitcoin_from_Source.md). * Si prefiere compilar Bitcoin Core a mano, eso se trata en el Apéndice 2.
+* [Usando GordianNode-macOS](https://github.com/BlockchainCommons/GordianNode-macOS). * Si tienes una Mac moderna, puedes usar la aplicación *GordianNode* de 
+Blockchain Commons, con tecnología *BitcoinStandup*, para instalar un nodo completo en su Mac.
+* [Uso de otros scripts de Bitcoin Standup](https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts). * Blockchain Commons también ofrece una versión del
```
[the remark this answers is not in the document]
Comment (javiervargas) [reply es/02_2_Configurando_Bitcoin_Core_Otros.md:12]: I dont see it.

**Review by csralvall — APPROVED** [review]
@shannona

**Pull request merged**